src/jit-analyze/jit-analyze.cs

```diff
@@ -997,7 +997,7 @@ public static void GenerateTSV(IEnumerable<FileDelta> compareList, string path)
                     // This should be split into:
                     //
                     // 32\t2\t/coreclr/bin/asm/asm/base/101301.dasm\t/coreclr/bin/asm/asm/diff/101301.dasm
-                    Regex gitMergedOutputRegex = new Regex(@"\{(\w+)\s=>\s(\w+)\}");
+                    Regex gitMergedOutputRegex = new Regex(@"\{((\w+(?:\/|\\)?)+)\s=>\s((\w+(?:\/|\\)?)+)\}");
```

Comment:
`git diff` has the `-z` command line option which will produce machine readable output (i.e. no =>, no escaped characters, things are separated by \0). Maybe it would be better to start using that to avoid more future problems?

Reply:
Thank you for the suggestion. I was not aware of `-z` option and it looks good. With that, I can get rid of some of the regexs and have simpler logic. The only downside is while debugging in VS, the output of `git diff` doesn't show properly in the debugger because of trailing `\0`, but I think that should be OK.

![image](https://user-images.githubusercontent.com/12488060/100553344-6f366c00-3242-11eb-8fee-3f0263e00259.png)